src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/Slider/SliderHandler.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace Xamarin.Platform.Handlers
+{
+	public partial class SliderHandler
+	{
+		public static PropertyMapper<ISlider, SliderHandler> SliderMapper = new PropertyMapper<ISlider, SliderHandler>(ViewHandler.ViewMapper)
+		{
+			[nameof(ISlider.Minimum)] = MapMinimum,
+			[nameof(ISlider.Maximum)] = MapMaximum,
+			[nameof(ISlider.Value)] = MapValue,
+			[nameof(ISlider.MinimumTrackColor)] = MapMinimumTrackColor,
+			[nameof(ISlider.MaximumTrackColor)] = MapMaximumTrackColor,
+			[nameof(ISlider.ThumbColor)] = MapThumbColor
+		};
+
+		public static void MapMinimum(SliderHandler handler, ISlider slider)
+		{
+			ViewHandler.CheckParameters(handler, slider);
+			handler.TypedNativeView?.UpdateMinimum(slider);
```

Comment:
The name of the extension methods should be:
- Update {Property}
- Set {Property}
?

Reply:
Update